WalletWasabi.Fluent/Views/AddWallet/WelcomePageView.axaml

```diff
@@ -150,8 +150,8 @@
         <!-- Trustless by design -->
         <Grid ColumnDefinitions="*,*">
           <DockPanel Grid.Column="0" Grid.Row="0" VerticalAlignment="Center">
-            <TextBlock Classes="text" Text="Being put in a position to trust someone is always uncomfortable. Especially regarding personal information or your money. This is why Wasabi Wallet is verifiably trustless by design. Don't trust, verify." />
-            <TextBlock Text="Trustless by design"
+            <TextBlock Classes="text" Text="If you don't control your private keys, then you are not the true owner of the funds. Wasabi is non-custodial: the developers cannot steal or freeze your funds." />
```

Comment:
Talking about private keys might confuse newbies since they only learn about seed words.
How about:

```suggestion
            <TextBlock Classes="text" Text="Wasabi Wallet is non-custodial, which means you are in 100% control of your funds and even the developers cannot steal or freeze your coins. Great power comes with a great responsibility so make sure you backup your wallet properly." />
```